test/test_notebook.py

```diff
@@ -100,6 +100,14 @@ def test_graph_modelling_language_using_select_speaker(save=False):
     run_notebook("agentchat_graph_modelling_language_using_select_speaker.ipynb", save=save)
 
 
+@pytest.mark.skipif(
+    skip or not sys.version.startswith("3.11"),
+    reason="do not run if openai is not installed or py!=3.10",
+)
+def test_groupchat_compression(save=False):
+    run_notebook("agentchat_groupchat_compression.ipynb", save=save)

```

Comment:
Should this notebook be run in the contrib-openai workflow instead of the openai workflow?

Reply:
Removing this test of notebook because the code is already tested in `test_compressible_groupchat.py`.